guides/user-manual/api_auth.rst

```diff
@@ -17,33 +17,53 @@ There a few options for authenticating against the M&M API:
 
 .. _api-login-command:
 
-Login command
+Login Command
 -------------
 
-This method is available in all versions of M&M, and all the different flavors of the M&M API (SOAP/JSON-RPC/REST). In the REST API, this command is available via the URL commands/Login
+This method is compatible with all versions of Micetro, and all every variant of the Micetro API (SOAP/JSON-RPC/REST). In the REST API, this command is accessible via the URL commands/Login
 
-The Login command takes in server (for the M&M Central server), user name, and password, and returns a session ID, which should then be used for all other API calls via their session parameter.
+The Login command requires a server (for the Micetro Central server), username, and password, and it returns a session ID. This session ID should be used for all other API calls via the session parameter.
 
-The Login command and its parameters is fully documented in the normal SOAP API documentation
+The Login command and its parameters are fully documented in the standard SOAP API documentation
 
 .. note::
-  Remember to configure HTTPS for the mmws and/or _mmwebext sites for this authentication method, since the username and password can easily be extracted for anyone listening in on the communication.
+  Remember to configure HTTPS for the mmws and/or _mmwebext sites when using this authentication method to prevent usernames and passwords from being intercepted during communication.
 
 .. _api-login-headers:
 
 Authorization Headers
 ---------------------
 
-With the introduction of Men&Mice Web Services and the REST API in version 7.3, it's now also possible to authenticate via HTTP headers, by adding a Basic Authorization header, or with a Negotiate Authorization header, if the client supports it.
+It's possible to authenticate via HTTP headers by adding a Basic Authorization header or a Negotiate Authorization header, if supported bye the client.
 
 .. note::
-  If AuthorizationMethods is not defined in the preference file mmWS will only accept basic authorization.
+  If AuthorizationMethods is not defined in the preference file, mmWS will only accept Basic Authorization.
```

Comment:
If AuthorizationMethods is not defined in the preference file, mmWS will accept Bearer and Basic Authorization headers.